service/web/server/docs/openapi.yaml

```diff
@@ -0,0 +1,623 @@
+                  type: number
+                  description: The new time step to set for the simulation.
+      responses:
+        '200':
+          description: Successful response indicating the time step was updated.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Simulation time step updated successfully."
+                  payload:
+                    type: object
+                    properties:
+                      step:
+                        type: number
+                        description: The new simulation time step that was set.
+                        example: 900
+                    required:
+                      - step
+                    description: Object containing the updated step value.
+                  status:
+                    type: integer
+                    example: 200
+
+  /forecast/{testid}:
+    parameters:
+      - $ref: '#/components/parameters/TestId'
+    put:
+      summary: Submit a forecast query
+      tags:
+        - data
+      description: Sends a ForecastQuery payload and returns a TimeSeries response.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ForecastQuery'
+      responses:
+        '200':
+          description: Successful response with forecast time series.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Forecast query processed successfully."
+                  payload:
+                    $ref: '#/components/schemas/TimeSeries'
+                  status:
+                    type: integer
+                    example: 200
+  /results/{testid}:
+    parameters:
+      - $ref: '#/components/parameters/TestId'
+    put:
+      summary: Submit a results query
+      tags:
+        - data
+      description: Sends a ResultsQuery payload and returns a TimeSeries response.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ResultsQuery'
+      responses:
+        '200':
+          description: Successful response with results time series.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Results query processed successfully."
+                  payload:
+                    $ref: '#/components/schemas/TimeSeries'
+                  status:
+                    type: integer
+                    example: 200
+  /kpi/{testid}:
```

Comment:
For the description, it would be good to copy what's in the user guide, since it adds some detail about what's included/not included in the kpi calculation.  See [here](https://ibpsa.github.io/project1-boptest/docs-userguide/api.html#get-kpi).

Also suggest creating a new schema that can be used to give explicit descriptions of the KPIs in the response that are returned as is done in the user guide [here](https://ibpsa.github.io/project1-boptest/docs-userguide/api.html#get-kpi).